larq/quantized_variable_test.py

```diff
@@ -21,7 +21,7 @@ def test_inheritance(distribute_scope):
     assert isinstance(quantized_variable, DistributedVariable) is distribute_scope  # type: ignore
 
 
-def test_read(distribute_scope, eager_and_graph_mode):
+def test_read(eager_and_graph_mode, distribute_scope):
```

Comment:
Why are these now all flipped?

Reply:
Due to the change above, we need to set the `graph_mode` fixture before setting a distribution scope, otherwise the cleanup will error due to the graph beeing clean up, before exiting the distribution scope.